Makefile

```diff
@@ -100,6 +101,13 @@ dist/subctl-%.tar.xz: cmd/bin/subctl-%
 	tar -cJf $@ --transform "s/^cmd.bin/subctl-$(VERSION)/" $<
 
 # Versions may include hyphens so it's easier to use $(VERSION) than to extract them from the target
+
+# Special case for Linux container builds
```

Comment:
Is there some more info that can be provided here (eg link) for why this special case is needed?

It's not clear why you need to pre create the dir, nor why youre re-linking `cmd/bin/subctl` at the end (as it's not the target of this build)

Reply:
See the comment in the updated PR. I’m not sure what you mean by “re-linking `cmd/bin/subctl` at the end”; this is linking the requested target to the listed prerequisite.

Reply:
OK gotcha, I thought it's overriding the link for `cmd/bin/subctl`